dropwizard-jobs-core/src/main/java/de/spinscale/dropwizard/jobs/JobManager.java

```diff
@@ -218,34 +225,58 @@ protected Trigger executeNowTrigger() {
         return TriggerBuilder.newTrigger().startNow().build();
     }
 
+    private void createScheduler() throws SchedulerException {
+        // backwards compatibility - only use this when something is provided in
+        // the config
+        if (!configuration.getQuartzConfiguration().isEmpty()) {
```

Comment:
if you put it the other way around
```java
if (configuration.getQuartzConfiguration().isEmpty()){
    scheduler = StdSchedulerFactory.getDefaultScheduler();
}
```
you avoid the double negation - I find that a lot easier to read.